### PR TITLE
[#40435] Update SAML documentation

### PR DESCRIPTION
**Pull request by ghost**
[#40435] Update SAML documentation

https://community.openproject.org/work_packages/40435

### CODE REVIEW

File: docs/system-admin-guide/authentication/saml/README.md [hunk 1]
```diff
@@ -132,6 +134,9 @@ Setting.plugin_openproject_auth_saml = Hash(Setting.plugin_openproject_auth_saml
       "name" => "saml",
       "display_name" => "My SSO",
       "assertion_consumer_service_url" => "https:/<YOUR OPENPROJECT HOSTNAME>/auth/saml/callback"
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/saml/README.md]: Quick reminder: You wanted to add the missing slash here

File: docs/system-admin-guide/authentication/saml/README.md [hunk 2]
```diff
@@ -61,11 +61,13 @@ saml:
   assertion_consumer_service_url: "https://<YOUR OPENPROJECT HOSTNAME>/auth/saml/callback"
   issuer: "https://<YOUR OPENPROJECT HOSTNAME>"
 
-  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here
+  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here in ONE line
+  ### one liner to generate certificate in ONE line
+  ### awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' <yourcert.pem>
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/saml/README.md:66]: :bow:

File: docs/system-admin-guide/authentication/saml/README.md [hunk 3]
```diff
@@ -171,6 +178,24 @@ You may provide attribute names or namespace URIs as follows: `email: ['http://s
...
+**b) Attribute mapping example for database**
+
+```ruby
+Setting.plugin_openproject_auth_saml = Hash(Setting.plugin_openproject_auth_saml).deep_merge({
+  "providers" => {
+    "saml" => {
+      "email" => "email",
+      "login" => "username",
+      "first_name" => "firstname",
+      "last_name" => "lastname"
```
Comment (oliverguenther) [inline docs/system-admin-guide/authentication/saml/README.md:190]: Even if it works, I'd add two attributes in an array here to show it is possible:

For example:
```
"login" => ['username', 'samAccountName', 'uid'],
```

**Review by oliverguenther — CHANGES_REQUESTED** [review]
Some minor remarks, mostly looking good thanks :+1:

File: docs/system-admin-guide/authentication/saml/README.md (hunk 2 above)
Comment (ghost) [reply docs/system-admin-guide/authentication/saml/README.md:66]: What does this emoji actually stand for? it seems to have no information to me... at least i cannot combine to anything else than a person with some green shirt

File: docs/system-admin-guide/authentication/saml/README.md (hunk 3 above)
Comment (ghost) [reply docs/system-admin-guide/authentication/saml/README.md:190]: HI @oliverguenther 
i added your suggested changes, please verify them
Thank you

**Review by oliverguenther — APPROVED** [review]

**Pull request merged**